SoftwareConstructionFundamental/SCF-Session9/Rectangle.java

```diff
@@ -0,0 +1,79 @@
+import java.text.DecimalFormat;
+import java.util.Date;
+
+/**
+ * This Rectangle class implements Shape interface
+ * @author Shubham Jain
+ *
+ */
+public class Rectangle implements Shape{
+
+	private Point origin;
+	private double height;
+	private double width;
+	private Date timeStamp;
+	DecimalFormat decimalFormatSpecifier = new DecimalFormat(".##");
+
+	public Rectangle(double height, double width, Point origin, Date timestamp) {
+		this.height = height;
+		this.width = width;
+		this.origin = origin;
+		this.timeStamp = new Date();
+		
+		/**
+		 * Checking if shape lies on the screen
+		 */
+		if ((origin.getXCoordinate() + width) > 1080 || (origin.getYCoordinate() + height) > 720) {
```

Comment:
@meta-shubham-jain 
There can be only one function in Point class which check pint in screen or not